ui/src/routes/guides/_layout.svelte

```diff
@@ -0,0 +1,197 @@
+<script>
+	import * as _ from 'lamb';
+	import {_screen}
+		from '@svizzle/ui/src/gauges/screen/ScreenGauge.svelte';
+
+	import ChevronLeft from '@svizzle/ui/src/icons/feather/ChevronLeft.svelte';
+	import ChevronRight from '@svizzle/ui/src/icons/feather/ChevronRight.svelte';
+	import Icon from '@svizzle/ui/src/icons/Icon.svelte';
+	import Link from '@svizzle/ui/src/Link.svelte';
+
+	import {isNotNil} from '@svizzle/utils';
+	import {toolName} from 'app/config';
+	import theme from 'app/theme';
+
+	const links = {
+		indicators: 'Indicators',
+		'a11ymenu': 'A11y menu'
```

Comment:
Apices round this key aren't needed

```suggestion
		a11ymenu: 'A11y menu'
```